src/main/riddl/Tenant/tenant.riddl

```diff
@@ -134,22 +36,31 @@ context TenantContext is {
                 meta: TenantContext.MetaInfo
             }
             handler DraftTenantHandler is {
-                on command ActivateTenant {
+                on command TenantContext.EditInfo {
+                    then "Copy fields from TenantContext.EditInfo into Draft info fields"
+                    and "adjust meta fields to reflect the change set"
+                    and "yield event TenantContext.InfoUpdated"
+                }
+                on command TenantContext.UpdateStatus {
                     example Success {
-                        then "Morph Tenant to Active state"
+                        then "Morph Tenant to desired state"
```

Comment:
The valid states for a Tenant are Draft, Active, and Suspended. We are in the draft handler. The state machine for a tenant moves from Draft -> Active <-> Suspended. Perhaps we need to define our messages as such. Or maybe in the DraftHandler if we try to move to suspended we throw an error? In other words, the only valid state transition from Draft is to Active.